cubewalkers/model.py

```diff
@@ -139,8 +138,9 @@ def __init__(
         self.initialize_walkers()
 
     def initialize_walkers(self) -> None:
-        """Generates initial conditions from internally stored data. See the
-        initial_conditions module for details.
+        """
+        Generates initial conditions from internally stored data. See the
+        `initial_conditions` module for details.
```

Comment:
Would making `initial_conditions` into `cubewalkers.initial_conditions` make it a hyperlink to the module? If so, it would be helpful. Also help the reader distinguish variables with refered modules. Same goes for `update_schemes`.

Reply:
To make it a hyperlink, you write it as `` :mod:`initial_conditions` `` for a module, or, e.g., `` :meth:`derrida_coefficient` `` for a method. In some simple testing, it seems to be able to figure out the correct scope in cases where we have the same function name in multiple modules (like in the `derrida_coefficient` example). I'll work on getting these links in today.

Reply:
Oh, and `:func:` for functions.

Reply:
One last thing to note: you can do aliases in this notation like this:
`` :class:`Model<cubewalkers.model.Model>` ``
Then the link appears as `Model` but will point to the right place.